os/drivers/audio/alc5658.c

```diff
@@ -1274,28 +1265,25 @@ FAR struct audio_lowerhalf_s *alc5658_initialize(FAR struct i2c_dev_s *i2c, FAR
 	/* Allocate a ALC5658 device structure */
 	priv = (FAR struct alc5658_dev_s *)kmm_zalloc(sizeof(struct alc5658_dev_s));
 	if (priv) {
+
 		/* Initialize the ALC5658 device structure.  Since we used kmm_zalloc,
 		 * only the non-zero elements of the structure need to be initialized.
 		 */
-
 		priv->dev.ops = &g_audioops;
 		priv->lower = lower;
 		priv->i2c = i2c;
 		priv->i2s = i2s;
 
 		sem_init(&priv->devsem, 0, 1);
 		dq_init(&priv->pendq);
-		dq_init(&priv->doneq);
```

Comment:
if we don't need doneq, why don't you remove doneq from structure in alc5658.h?(Actually I think we don't need it)

Reply:
Already removed in .h file also as part of this commit itself.